addons/project_todo/views/project_task_views.xml

```diff
@@ -86,7 +86,7 @@
                 <field name="priority" widget="priority" nolabel="1"/>
                 <field name="state" widget="todo_done_checkmark" nolabel="1"/>
                 <field name="name"/>
-                <field name="user_ids" optional="show" widget="many2many_avatar_user" options="{'no_quick_create': True}"/>
+                <field name="user_ids" optional="show" required="1" widget="many2many_avatar_user" options="{'no_quick_create': True}"/>
```

Comment:
Why is the 'user_ids' field required only for the list view, and what about form view?